ui/src/tool-panel/location-track/splitting/location-track-splitting-infobox.tsx

```diff
@@ -457,6 +468,27 @@ export const LocationTrackSplittingInfobox: React.FC<LocationTrackSplittingInfob
                                         </MessageBox>
                                     </InfoboxContentSpread>
                                 )}
+                                {firstChangedDuplicateInSplits && (
+                                    <InfoboxContentSpread>
+                                        <MessageBox type={'ERROR'}>
+                                            <div>
+                                                {t(
+                                                    'tool-panel.location-track.splitting.validation.duplicate-not-published',
+                                                    {
+                                                        duplicateName:
+                                                            firstChangedDuplicateInSplits.name,
+                                                    },
+                                                )}
+                                            </div>
+                                            <br />
+                                            <div>
+                                                {t(
+                                                    'tool-panel.location-track.splitting.validation.publish-duplicate',
+                                                )}
+                                            </div>
+                                        </MessageBox>
+                                    </InfoboxContentSpread>
+                                )}
```

Comment:
Meinaa tulla aika reippaasti rivitystä runsaan sisennyksen vuoksi tässä, komponentti?

Reply:
Ajattelin käydä splittausinfoboksinkin läpi samalla kun käyn läpi tuon koko `LocationTrackInfobox`:in myöhemmin.